### PR TITLE
Add sanity test and changelog workflow

### PR DESCRIPTION
**Pull request by GomathiselviS**
This PR adds test workflows to Github Actions for this collection.

### CODE REVIEW

File: plugins/modules/eval_nat_network_acls.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
     description:
     - The port range used by the AWS resource in your Amazon VPC you want to test connectivity from.
     type: str
-    required: false
+    required: true
```
Comment (alinabuzachis) [inline plugins/modules/eval_nat_network_acls.py:36]: It should be optional, so required: false. Please update in the argument spec also.

Comment (GomathiselviS) [reply plugins/modules/eval_nat_network_acls.py:36]: Removing the required field should work. I shall update the arg spec accordingly.

**Comment by alinabuzachis** [comment]
@GomathiselviS This https://github.com/redhat-cop/cloud.aws_troubleshooting/pull/22 should fix some of those sanity errors.

File: galaxy.yml [hunk 2]
```diff
@@ -7,15 +7,15 @@ authors:
...
-  community.aws: '>=5.4.0'
 tags:
   - amazon
   - aws
   - cloud
   - infrastructure
   - cluster
+dependencies:
+  amazon.aws: '>=5.1.0'
+  community.aws: '>=5.0.0'
```
[the remark this answers is not in the document]
Comment (GomathiselviS) [reply galaxy.yml:18]: The galaxy.yml file was missing the dependencies. So I used the values from [cloud.aws_ops galaxy.yml](https://github.com/redhat-cop/cloud.aws_ops/blob/main/galaxy.yml#L19)

[the remark this answers is not in the document]
Comment (GomathiselviS) [reply galaxy.yml:18]: Also, we are not downgrading the dependencies. The latest one available on galaxy will be picked up.

**Review by gravesm — APPROVED** [review]

**Pull request merged**